### DISCUSSION TITLE
Mirth's channel which connect to the db via java stopped working after installing this on the server Microsoft KB 5037016

### DISCUSSION THREAD

**Discussion by lilianamSF** [nextgenhealthcare/connect#6367] [Q&A]
Would anyone in this forum have an idea why would Mirth's channel which connect to the db via java stopped working after installing the update below?

[https://support.microsoft.com/en-us/topic/kb5037016-servicing-stack-update-for-windows-10-version-1607-and-server-2016-april-9-2024-fa80a866-2625-4f46-8f51-bffed6e7887e]

**Comment by pacmano1** [comment]
What does "via java" mean?   You need to ask a more thorough question.    Is this a database reader? writer?  A connection built manually in javascript via jdbc?   What error are you getting.   While this is not the mirth forums, please read https://forums.mirthproject.io/articles/177082-forum-etiquette to understand how to ask questions.

**Reply by lilianamSF** [reply]
Hi pacmano, thank you for your guidance.

- We are using Mirth 4.4.2 and Java 17.
- The Mirth channel is a database reader
- The error we are getting is:

`[2024-11-18 17:21:10.010]  ERROR  (com.mirth.connect.connectors.jdbc.DatabaseReceiverScript:192): An error occurred while polling for messages, retrying after 10000 ms... 
java.lang.NullPointerException 
at com.mirth.connect.connectors.jdbc.DatabaseReceiverScript.poll(DatabaseReceiverScript.java:188) 
at com.mirth.connect.connectors.jdbc.DatabaseReceiver.poll(DatabaseReceiver.java:140) 
at com.mirth.connect.donkey.server.channel.PollConnectorJob.execute(PollConnectorJob.java:49) 
at org.quartz.core.JobRunShell.run(JobRunShell.java:213) 
at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:557)`
